src/index.js

```diff
@@ -177,48 +180,101 @@ class Saturn {
   }
 
   async _reportLogs () {
-    if (this.logs.length) {
-      await fetch(
-        'https://twb3qukm2i654i3tnvx36char40aymqq.lambda-url.us-west-2.on.aws/',
-        {
-          method: 'POST',
-          body: JSON.stringify({
-            bandwidthLogs: this.logs
-          })
-        }
-      )
-      this.logs = []
+    if (!this.logs.length) {
+      return
+    }
+
+    let bandwidthLogs = this.logs
+    if (this.hasPerformanceAPI) {
+      bandwidthLogs = this._matchLogsWithPerformanceMetrics(bandwidthLogs)
     }
+
+    await fetch(
+      this.opts.logURL,
+      {
+        method: 'POST',
+        body: JSON.stringify({ bandwidthLogs, logSender: this.opts.logSender })
+      }
+    )
+
+    this.logs = []
+    this._clearPerformanceBuffer()
```

Comment:
Why is the performance buffer cleared everytime we report logs?

Reply:
It will eventually fill up to the max size, then entries will start being dropped. So we clear it to prevent dropped entries.
The assumption here is it's safe to clear as we've just processed all saturn requests in the current buffer, though that's not guaranteed.

Reply:
Do you think it would be better to treat the buffer as a queue where we evict old entries with new ones instead of clearing the whole buffer or is that too memory intensive to keep the buffer at max size the whole time?

Reply:
The browser populates the buffer. AFAIK the only thing users can do to manage it is clear it, or increase the size. 

What we can do is monitor the buffer and extract saturn request into our own managed buffer. But I feel like the current implementation is "good enough" to where we don't need the extra complexity. Though not sure until we see it in action.

Reply:
Sounds good, guess we can revisit if needed